package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/cdk",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
Whoops! This is because I branched off of main locally but the release PR hasn't been merged yet. Will rebase once that's approved and merged.